src/Features/Core/Portable/Completion/CompletionService_GetCompletions.cs

```diff
@@ -404,7 +404,7 @@ public void Add(CompletionItem item)
 
                     Count++;
                     // Matching items should be rare, no need to use object pool for this.
-                    _displayNameToItemsMap[entireDisplayText] = new List<CompletionItem>() { sameNamedItem, item };
+                    _displayNameToItemsMap[entireDisplayText] = new List<CompletionItem>() { item, sameNamedItem };
```

Comment:
We probabaly don't want to rely on the implementation detail like this to determine the item order. Given the order is ultimatedly decided by `SortText`, consider using it to differentiate two items instead.